migration/migrator/migrations/course/20190628100043_create_rainbow_grades_dir.py

```diff
@@ -0,0 +1,28 @@
+import os
+import grp
+from pathlib import Path
```

Comment:
These migration files should probably be combined.

Reply:
this PR will be merged after another PR which introduces these migrations.  
But yes, I agree, the migrations should be combined (and re-numbered to be 201907XX).
And then removed from this PR.